threads-web-ui/next.config.js

```diff
@@ -4,7 +4,7 @@ const nextConfig = {
     remotePatterns: [
       {
         protocol: 'https',
-        hostname: '**.cdninstagram.com',
+        hostname: '**.cdninstagram.com, **.fbcdn.net',
```

Comment:
I dont think this will work

Reply:
<img width="1076" alt="image" src="https://github.com/junhoyeo/threads-api/assets/32605822/b8e407ac-f717-45f4-979d-89542ac7ba8b">
See